frontend/src/components/Patchnotes.jsx

```diff
@@ -20,9 +20,25 @@ const Patchnotes = ({ game }) => {
     },
     comments: [],
   });
+  const [associations, setAssociations] = useState({ nerf: [], buff: [] });
   const [error, setError] = useState(null);
   const [modalOpen, setModalOpen] = useState(false);
   const [currentReply, setCurrentReply] = useState({ message: '', replyToId: null, commentId: null, parentReplyId: null });
+  const [filter, setFilter] = useState('all');
+
+  const classifyUpdate = (text) => {
+    for (let keyword of associations.nerf) {
+      if (text.toLowerCase().includes(keyword[0].toLowerCase()) && text.toLowerCase().includes(keyword[1].toLowerCase())) {
+        return 'nerf';
```

Comment:
even if these strings are being used once, they should be saved in const/static variables for 1) readability and 2) future improvements/changes